development-guide.md

```diff
@@ -180,6 +180,13 @@ Also, Fivetran deduplicates operations such that each primary key shows up only
     - `null_string` value is used to represent `NULL` value in all batch files.
     - `unmodified_string` value is used to indicate columns in `update_files` where the values did not change.
 
+#### WriteHistoryBatchRequest
+The `WriteHistoryBatchRequest` RPC call provides details about the batch files containing the records to be pushed to the destination for **History Mode**. In addition to the parameters of `WriteBatchRequest`, this request also contains following batch file
```

Comment:
```suggestion
The `WriteHistoryBatchRequest` RPC call provides details about the batch files containing the records to be pushed to the destination for **History Mode**. In addition to the parameters of `WriteBatchRequest`, this request also contains the `earliest_start_files` file used for updating history mode-specific columns for the existing rows in the destination.
```

Reply:
@shivangisingh172 
1. I believe the history mode is enabled on table level maximum, why do we use "history mode-specific columns" here?
2. Why is the file  called `earliest_start_files` since it's a single file? Does it contain information for multiple files in the source?
3. I would advise against using the same name for method and file - adds confusion and requires specifying the entity name in each instance.